main.js

```diff
@@ -75,22 +80,39 @@ async function acceptStudent(db, ID, studentName, studentAge, studentAddress) {
     };
     return await db.put(ID, studentInfo);
 }
+
 async function scheduleInterview(db, ID, ScheduleDate) {
     try {
         const student =  await db.get(ID);
         student.interviewSched = ScheduleDate;
         student.status = 'Under Interview';
-        await db.put(ID, student);
+        db.put(ID, student);
     } catch (error) {
         console.log('The ID', ID ,'you entered is not existing');
     }
 }
+
 async function scheduleExam(db, ID, ScheduleDate){
     try {
         const student = await db.get(ID);
         student.examdSched = ScheduleDate;
         student.status = 'Exam Pending';
-        await db.put(ID, student);
+        db.put(ID, student);
+    } catch (error) {
+        console.log('The ID', ID ,'you entered is not existing');
+    }
+}
+
+async function rateEntranceExam(db, ID, examScore){
+    try {
+        const student = await db.get(ID);
+        student.score = examScore;
+        if(examScore >= 70){
+            student.status = 'Admitted';
+        } else{
+            student.status = 'Probationary';
+        }
+        db.put(ID, student);
```

Comment:
await is need when calling db.put
```suggestion
        await db.put(ID, student);
```